README.md

```diff
@@ -1,127 +1,187 @@
-## GraphStorm
-| [Document and Tutorial Site](https://graphstorm.readthedocs.io/en/latest/) | [GraphStorm Paper](https://arxiv.org/abs/2406.06022) |
+## GraphStorm: Enterprise graph machine learning framework for billion-scale graphs
+
+[![PyPI version](https://badge.fury.io/py/graphstorm.svg)](https://badge.fury.io/py/graphstorm)
+[![CI Status](https://github.com/awslabs/graphstorm/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/awslabs/graphstorm/actions/workflows/continuous-integration.yml)
+[![Docs Status](https://app.readthedocs.org/projects/graphstorm/badge/?version=latest)](https://graphstorm.readthedocs.io/en/latest/)
+
+| [Documentation and Tutorial Site](https://graphstorm.readthedocs.io/en/latest/) | [GraphStorm Paper](https://arxiv.org/abs/2406.06022) |
```

Comment:
Will it duplicate with the above Docs Status?

Reply:
So we will have three badges, right?

Reply:
Yes 3 badges that show latest version on pypi, CI health , and docs health.

Reply:
Here's how they look when rendered: https://github.com/thvasilo/graphstorm/blob/readme-updates/README.md